include/triton/common/triton_json.h

```diff
@@ -109,6 +110,19 @@ class TritonJson {
     std::string buffer_;
   };
 
+  static std::string EscapeString(const std::string& input)
+  {
+    WriteBuffer writebuffer;
+    const unsigned int writeFlags = rapidjson::kWriteNanAndInfFlag;
+    // Provide default template arguments to pass writeFlags
+    rapidjson::Writer<
+        WriteBuffer, rapidjson::UTF8<>, rapidjson::UTF8<>,
+        rapidjson::CrtAllocator, writeFlags>
+        writer(writebuffer);
+    writer.String(input.c_str());
```

Comment:
I was going to ask if this can fail if the input has some malformed or half-formed JSON in it, but it seems like it won't: https://github.com/Tencent/rapidjson/issues/1767#issuecomment-672541097

Reply:
error handling was a little tricky w.r.t to we usually log an error - so I didn't want to create a recursion by accident. Writing and error causes an error .... so would probably have to fail silently ... or print directly to the stdout ...